indico/core/db/sqlalchemy/util/models.py

```diff
@@ -132,13 +132,17 @@ def assign_id(self):
             id_ = db.session.query(db.func.nextval(db.func.pg_get_serial_sequence(table_name, col_name))).one()[0]
         setattr(self, attr_name, id_)
 
-    def populate_from_dict(self, data):
+    def populate_from_dict(self, data, skip=None):
         """Populates the object with values in a dictionary
 
+        if `skip`is set, keys in that list are skipped.
```

Comment:
missing space, but i'd document it as a param, i.e.

    :param skip: If set, fields in that list are skipped

(i know it's not like that in IndicoForm.populate_obj and eventually that docstring should be rewritten)